tests/base/deriving_foldable/DeriveFoldable.idr

```diff
@@ -57,7 +57,7 @@ namespace BigTree
 
 namespace Matrix
 
-  record Matrix m n a where
+  record Matrix (m, n : Nat) (a : Type) where
```

Comment:
Did you change the quantity of those `m`, `n` and `a` here?

Reply:
No, untyped record arguments are assigned `top` in the parser.

Reply:
OMG, rules for records and interfaces differ! Maybe, it's good that it can be (accidentally) removed by this PR